retux/client/resources/channel.py

```diff
@@ -315,7 +349,39 @@ class MessageActivity:
     """The ID of a party from a rich presence event."""
 
 
-@define()
+@define(kw_only=True)
+class MessageReference:
+    """
+    Represents a reference to a message from Discord.
+    
+    Attributes
+    ----------
+    message_id : `Snowflake`, optional
+        The ID of the originating message.
+    channel_id : `Snowflake`, optional
+        The ID of the originating message's channel.        
+    guild_id : `Snowflake`, optional
+        The ID of the originating message's guild.
+    fail_if_not_exists : `bool`
+        Whether or not to error if the referenced doesn't exist.
+    
+        If `False`, sends as a normal, non-reply message.
+    """
+    message_id: str | Snowflake | None = field(converter=optional_c(Snowflake), defualt=None)
+    """The ID of the originating message."""
+    channel_id: str | Snowflake | None = field(converter=optional_c(Snowflake), defualt=None)
+    """The ID of the originating message's channel."""
+    guild_id: str | Snowflake | None = field(converter=optional_c(Snowflake), defualt=None)
+    """The ID of the originating message's guild."""
+    fail_if_not_exists: bool | None = field(default=True)
```

Comment:
```suggestion
    fail_if_not_exists: bool = field(default=True)
```
The `None` field can be removed since we're setting a default of the boolean.